### ISSUE TITLE
AOSP and GPS...

### ISSUE THREAD

**Issue by nikolowry** [mapsme/omim#2041]
I've been using maps.me for a few months on and off - but always sem to ditch it because it can never detect my GPS coordinates.  SoI headed over to your site's help section and stumbled onto this blurb:

>_I use the Android version of MAPS.ME and cannot determine my location._
Check your Geolocation Settings and make sure that Google Play Services are not blocked. We use these to determine your location, but they may be blocked by some antivirus services, so check these settings too. Try to enable (disable if enabled) “Use Google Play Services” in app settings, then restart your device.

I know you have recently rebranded - so I was wondering if this is still accurate to the current codebase (too much to skim through by myself right now)?

If it is, has there been any discussion to ditch the dependence of Google Play? As someone who tries to only use FOSS software(wish it was possible w hardware too), it be amazing to find an actively developed map app that positions itself as a real competitor to Google Maps on non-google devices.

If ridding the dependencies is too difficult/time consuming - could you atleast add an option for users to manually position the "current location". I'd really love to use this for navigation, but the apps useless when location is unknown.

Device: Moto X Style (Pure)
OS: AOSP MM + Moto Modem Firmware

![screenshot_20160226-222448](https://cloud.githubusercontent.com/assets/597310/13370530/2d7b443a-dcd8-11e5-8e5a-62ee319aaae0.png)

**Comment by nikolowry** [comment]
By installing LocalWifiNlpBackend I was able to immediately get a current location.

It's available on FDroid - https://f-droid.org/repository/browse/?fdfilter=Gps&fdid=org.fitchfamily.android.wifi_backend 

However it be nice to have some documentation for those of us who prefer a Google-free experience

**Comment by deathbaba** [comment]
Have you tried to turn off "use google play services" setting in our maps Settings menu?

**Comment by nikolowry** [comment]
Yep and actually sorry about the rant. By setting up microGsmCore and UnifedNip backends - the maps have been working flawlessly
